### ISSUE TITLE
Error with compiled ODD

### ISSUE THREAD

**Issue by scornec** [music-encoding/customeization#8]
Hello,

I try to use the MEI Customization service, but i get an error.
I choose the compiled ODD as output format, i try different MEI Release and MEI Customization file, but i get an error when i try to download it. 
My Firefox says that the file /tmp/mozilla_segolene0/z74F3wJc.xml.part cant be registered because the source file cant be read (the original message is in french).

Transformation Command: ['/usr/local/share/tei/Stylesheets/bin/teitoodd', '--localsource=/tmp/tmpk2py528w/source/driver.xml', '/tmp/tmpk2py528w/customizations/mei-CMN.xml', '/tmp/tmpuvuz1hmq/mei-CMN.xml']
Output: using /tmp/tmpk2py528w/source/driver.xml as default source
Convert mei-CMN.xml to /tmp/tmpuvuz1hmq/mei-CMN.xml (tei to odd) using profile default
     [echo] Do ODD expand processing (schema ) 

BUILD SUCCESSFUL
Total time: 4 seconds

Thank you in advance for your help

Ségolène

**Comment by ahankinson** [comment]
Yes, we're aware of the problem with the customization service and are working on a fix. Thanks for your patience.

**Comment by ahankinson** [comment]
This should be fixed now.

**Comment by bwbohl** [comment]
@ahankinson can this be closed?

**Comment by musicEnfanthen** [comment]
Closed, because this service is deprecated and the repo has been archived.
